youtubeBot/bot.py

```diff
@@ -32,8 +34,26 @@ def send_text(self, text):
 
 
 class YoutubeBot(Bot):
-    pass  # TODO your code here!
+    """dictionary to save the songs and their names"""
+    exist_video = {}
```

Comment:
This variable should be in __init__ function (in the constructor) as a class attribute